vllm/v1/worker/gpu_model_runner.py

```diff
@@ -371,6 +403,18 @@ def execute_model(
         return model_runner_output
 
     def load_model(self) -> None:
+        if self.use_cuda_graph:
+            # FIXME(woosuk): Currently, the custom ops are not supported
+            # in the piecewise compilation mode. We rely on TorchInductor
+            # to optimize the model.
+            envs.VLLM_CUSTOM_OPS = "none"
```

Comment:
iirc, `envs` does not support setting env vars.

Reply:
Good catch. Fixed.